src/Messenger.js

```diff
@@ -209,13 +209,40 @@ function sendFBMessage(userId, message, isMessageSentFromCheckIn) {
   });
 }
 
-function sendSMSMessage(userId, message) {
-  const twilioMessage = Object.assign({
-    from: process.env.TWILIO_NUMBER,
-    to: userId
-  }, message);
-  const twilioPromise = twilioClient.messages.create(twilioMessage);
-  return twilioPromise;
+/**
+ * This functions sends SMS to the client from the bot via twilio
+ * @param {number} userId the phone number of the user
+ * @param {object} message the message object
+ * @param {number} orgId The ID of the organization
+ * @returns twilio promise
+ */
+async function sendSMSMessage(userId, message, orgId) {
+  const {
+    TWILIO_ACCOUNT_SID,
+    TWILIO_AUTH_TOKEN,
+    TWILIO_NUMBER,
+    NODE_ENV,
+  } = process.env;
+  const orgCredentials = await api.getOrg(orgId);
+  let twilioAccountSid = orgCredentials.account_sid;
+  let twilioAuthToken = orgCredentials.auth_token;
+  let twilioPhoneNumber = orgCredentials.twilio_number;
+  
+  if (NODE_ENV === 'development'
```

Comment:
I think a single if check would work here.

```
if (NODE_ENV !== production) {
  // continue
}
```

Reply:
You're right @65black. Thank you!